pkg/apis/apis/v1alpha1/types.go

```diff
@@ -251,6 +267,26 @@ type APIBindingList struct {
 	Items []APIBinding `json:"items"`
 }
 
+// These are valid conditions of APIExport.
+const (
+	APIExportIdentityValid conditionsv1alpha1.ConditionType = "IdentityValid"
+
+	IdentityVerificationFailedReason = "IdentityVerificationFailed"
+	IdentityGenerationFailedReason   = "IdentityGenerationFailed"
+)
+
+// These are for APIExport identity.
+const (
+	// AnnotationKeyAPIExportCluster is the key for an annotation on an identity secret for an APIExport for the
+	// cluster name for the associated APIExport.
+	AnnotationKeyAPIExportCluster = "apis.kcp.dev/apiexport-cluster"
+	// AnnotationKeyAPIExportName is the key for an annotation on an identity secret for an APIExport for the
+	// name for the associated APIExport.
+	AnnotationKeyAPIExportName = "apis.kcp.dev/apiexport-name"
```

Comment:
does this mean we cannot share the identity secret between exports?

Reply:
That wasn't the intent, after chatting with you a couple of weeks ago. I need to undo this.

Reply:
Fixed